Tests/SpeziDevicesTests/HealthMeasurementsTests.swift

```diff
@@ -10,19 +10,21 @@ import HealthKit
 @_spi(TestingSupport) import SpeziBluetooth
 import SpeziBluetoothServices
 @_spi(TestingSupport) @testable import SpeziDevices
-import XCTest
+import Testing
 
 
-final class HealthMeasurementsTests: XCTestCase {
+@Suite
+final class HealthMeasurementsTests {
     @MainActor
+    @Test
     func testReceivingWeightMeasurements() async throws {
```

Comment:
we should, in order to better match the Swift Testing naming conventions, remove the `test` prefix from all test functions.

```suggestion
    func receiveWeightMeasurements() async throws {
```